server/src/main/java/org/elasticsearch/index/flush/FlushStats.java

```diff
@@ -23,6 +24,7 @@ public class FlushStats implements Writeable, ToXContentFragment {
     private long total;
     private long periodic;
     private long totalTimeInMillis;
+    private long totalTimeExcludingWaitingInMillis;
```

Comment:
without *waiting* for what exactly? is it for commit durability? I miss a bit better naming here. wdyt?